### PR TITLE
SOF-1235 Change Skema mapping in Blueprint config

### PR DESCRIPTION
**Pull request by RasmusAlbrektsen**
The old Skema is replaced by the new GFX Skema Templates. It should be noted that the acceptance criteria:
"If an entry has both an iNews name and iNews column, the iNews column takes priority." has not been considered, since this isn't relevant or possible to do until further implementation later in the phase regarding Overlay Mapping.

Maybe we should have a conversation about the naming of config tables as well. Especially how "Graphics Setups" should probably be "Graphic Setups", and that we use "Skema" in the user facing strings rather than the english name.

### CODE REVIEW

**Review by LindvedKrvang — APPROVED** [review]

File: src/tv2_afvd_showstyle/config-manifests.ts [hunk 1]
```diff
@@ -253,17 +245,17 @@ export const gfxDesignTemplates: ConfigManifestEntry[] = [
...
+				name: 'iNews Skema Column',
+				description: 'The selected skema',
 				type: ConfigManifestEntryType.STRING,
 				required: false,
 				defaultVal: '',
 				rank: 1
 			},
 			{
 				id: 'VizTemplate',
-				name: 'GFX Template Name',
```
Comment (ianshade) [inline src/tv2_afvd_showstyle/config-manifests.ts:266]: I believe that wherever, 'Viz Template Name' appears, we were going to change it to 'GFX Template Name' instead, and have a more detailed description (diferent between Qbox and Gallery showstyles). At least this is the way it was done in the two other GFX tables.

File: src/tv2_offtube_showstyle/config-manifests.ts [hunk 2]
```diff
@@ -182,6 +185,49 @@ export const gfxDesignTemplates: ConfigManifestEntry[] = [
...
+				name: 'iNews Skema Column',
+				description: 'The selected skema',
+				type: ConfigManifestEntryType.STRING,
+				required: false,
+				defaultVal: '',
+				rank: 1
+			},
+			{
+				id: 'VizTemplate',
+				name: 'Viz Template Name',
```
Comment (ianshade) [inline src/tv2_offtube_showstyle/config-manifests.ts:220]: There's no Viz in Qboxes, and I'm not sure what skema corresponds to in the HTML package, but if it is or will be used, the description could at least say something like `The name of the skema in the HTML package`

**Review by ianshade — COMMENTED** [review]
Again I'm stirring the pot with some Viz-related naming

File: src/tv2_offtube_showstyle/config-manifests.ts (hunk 2 above)
Comment (RasmusAlbrektsen) [reply src/tv2_offtube_showstyle/config-manifests.ts]: Updated! The text, however, uses Jacobs suggestion from Slack.

**Comment by RasmusAlbrektsen** [comment]
Changed target branch from develop to staging so it can be included in release 46.1.

**Review by ianshade — APPROVED** [review]
The new changes look good!

**Pull request merged**